src/main/java/net/javadiscord/javabot/systems/moderation/AutoMod.java

```diff
@@ -30,6 +30,11 @@
 public class AutoMod extends ListenerAdapter {
 
 	private static final Pattern inviteURL = Pattern.compile("discord(?:(\\.(?:me|io|gg)|sites\\.com)/.{0,4}|app\\.com.{1,4}(?:invite|oauth2).{0,5}/)\\w+");
+	Pattern urlPattern = Pattern.compile(
```

Comment:
Why is this package private?

Reply:
I meant you may want to make it private, not make other variables package private, too.

Reply:
alright